app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -68,4 +72,9 @@ class AboutActivity : BaseNavigationActivity() {
             Toast.makeText(this, R.string.no_webbrowser_available, Toast.LENGTH_SHORT).show()
         }
     }
+
+    @Suppress("UNUSED_PARAMETER")
+    fun showWhatsNew(v: View) {
+        _whatsNew.showDialog()
```

Comment:
It seems that there is no need for `_whatsNewInfo` to be a class member. I would simply do `WhatsNewInfo(this).showDialog()` and remove the member variable.